test/sqllogictest/transform/dataflow.slt

```diff
@@ -62,8 +62,12 @@ EXPLAIN PLAN FOR VIEW foo3
 | ArrangeBy (#0)
 
 %1 =
-| Join %0
+| Constant (6)
+
+%2 =
+| Join %0 %1 (= #0 #1)
 | | implementation = IndexedFilter #0 = 6
+| Project (#0)
```

Comment:
we need to align on the name of this join type - the variant is called `PredicateIndex`, the output here is `IndexedFilter`...

Reply:
Yes, I've changed the name of the join implementation to `IndexedFilter`.